build.gradle

```diff
@@ -13,7 +13,9 @@ buildscript {
     }
 }
 
+apply plugin: 'base'
 apply from: "${rootProject.buildDir}/release/gradle/version.gradle"
+apply from: "${rootProject.buildDir}/release/gradle/release.gradle"
```

Comment:
Or even better it should be after `version = scmVersion.version` (the second line).